.github/workflows/java11.yml

```diff
@@ -0,0 +1,116 @@
+name: Workflow to build and test GenomicsDB with Java 11
```

Comment:
Why a separate workflow here instead of trying to add Java11 to the `basic` matrix?

Reply:
I'm testing an attempt at using the matrix functionality here: https://github.com/GenomicsDB/GenomicsDB/commit/af83a205938bf435e1256654366c2957c3db7664

I'll merge it in here if it works.

Reply:
Ok I've updated this, integrated from https://github.com/GenomicsDB/GenomicsDB/actions/runs/494877899